firefox-ios/Client/Frontend/Browser/Toolbars/Redux/AddressBarState.swift

```diff
@@ -23,6 +24,7 @@ struct AddressBarState: StateType, Sendable, Equatable {
     var shouldSelectSearchTerm: Bool
     var isLoading: Bool
     let readerModeState: ReaderModeState?
+    let canSummarize: Bool?
```

Comment:
Does this really need to be optional? We only ever check if `canSummarize` is true.